docs/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## NEXT
+- Add the `Renderable` protocol, a consistent API to render an `htpy` object as HTML or to iterate over it. `Element`, `Fragment`, `ContextProvider`, and `ContextConsumer` are all `Renderable`.
+- Deprecate `render_node()` and `iter_node()` and direct iteration over elements. Call `Renderable.__str__()` or `Renderable.iter_chunks()` instead. [Read the Usage docs for more details](usage.md#renderable).
+
 ## 25.4.0 - 2025-04-10
```

Comment:
Missing #96 ;-)

Reply:
oops! will try to make sure changelogs are always part of PRs going forward so I do not forget. Fixed in main+updated docs+github release.